s3/pkg/datastore/aws/aws.go

```diff
@@ -43,26 +43,26 @@ import (
 )
 
 type AwsAdapter struct {
-	backend *backendpb.BackendDetail
-	session *session.Session
+	Backend *backendpb.BackendDetail
```

Comment:
As these are not SONY ODA specific changes, these fmt changes can be done as part of separate optimization PR